Description: Fix FTBFS against libffi 3.3
Author: Matthias Klose <doko@debian.org>
Bug-Debian: https://bugs.debian.org/949288
Applied-Upstream: https://gitlab.com/embeddable-common-lisp/ecl/commit/b2f09b4809441a92d6c11a2b39d5399580e56ae7
Last-Update: 2020-01-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/c/ffi.d
+++ b/src/c/ffi.d
@@ -132,10 +132,13 @@ static struct {
   {@':stdcall', FFI_STDCALL},
 #elif defined(X86_WIN64)
   {@':win64', FFI_WIN64},
-#elif defined(X86_ANY) || defined(X86) || defined(X86_64)
+#elif defined(X86_64)
+  {@':unix64', FFI_UNIX64},
+  {@':cdecl', FFI_UNIX64},
+  {@':sysv', FFI_UNIX64},
+#elif defined(X86_ANY) || defined(X86) 
   {@':cdecl', FFI_SYSV},
   {@':sysv', FFI_SYSV},
-  {@':unix64', FFI_UNIX64},
 #endif
 };
 
